Resources/Prototypes/Entities/Mobs/NPCs/slimes.yml

```diff
@@ -111,18 +105,58 @@
     successChance: 0.5
     interactSuccessString: petting-success-slimes
     interactFailureString: petting-failure-generic
+  - type: Speech
+    speechVerb: Slime
+    speechSounds: Slime
+  - type: TypingIndicator
+    proto: slime
+
+- type: entity
+  name: basic slime
+  id: MobAdultSlimes
+  parent: BaseMobAdultSlimes
+  abstract: true
+  description: It looks so much like jelly. I wonder what it tastes like?
+  components:
   - type: ReplacementAccent
     accent: slimes
   - type: GhostTakeoverAvailable
   - type: GhostRole
     makeSentient: true
     name: ghost-role-information-slimes-name
     description: ghost-role-information-slimes-description
-  - type: Speech
-    speechVerb: Slime
-    speechSounds: Slime
-  - type: TypingIndicator
-    proto: slime
+  - type: NpcFactionMember
+    factions:
+    - SimpleNeutral
+  - type: HTN
+    rootTask:
+      task: SimpleHostileCompound
+
+- type: entity
+  name: geras # hahahahahhaha geras old age and you;re jlike devolving into a baby version so funny i know
+  description: A geras of a slime - the name is ironic, isn't it?
+  id: MobSlimesGeras
+  parent: BaseMobAdultSlimes
+  noSpawn: true
+  components:
+  # they portable...
+  - type: MultiHandedItem
+  - type: Item
+    size: Huge
+  - type: Sprite
+    color: "#FFFFFF55"
+  - type: MeleeWeapon
+    attackRate: 2
+    damage:
+      types:
+        Blunt: 8
+        Structural: 5
```

Comment:
Slimes shouldn't be able to break out of perma as you'd have to redesign every single map.